django/http/request.py

```diff
@@ -230,6 +288,17 @@ def body(self):
         if not hasattr(self, '_body'):
             if self._read_started:
                 raise RawPostDataException("You cannot access body after reading from request's data stream")
+
```

Comment:
Exactly, it has its own logic. Therefore it does not access request,body in order not to read the whole request into memory. Or am I wrong? :)